corehq/apps/export/models/new.py

```diff
@@ -120,7 +119,7 @@ class ExportColumn(DocumentSchema):
     # Determines whether or not to show the column in the UI Config without clicking advanced
     is_advanced = BooleanProperty(default=False)
     selected = BooleanProperty(default=False)
-    tags = ListProperty()
+    tags = ListProperty()  # TODO: I think this is moving to SystemExportColumn?
```

Comment:
Yeah, I think I'm actually just going to leave them here. I also didn't end up creating a `SystemExportColumn` anyways.